cherrypy/test/test_logging.py

```diff
@@ -199,7 +199,7 @@ def test_custom_log_format(log_tracker, monkeypatch, server):
 
 
 def test_utc_in_timez(monkeypatch):
-    """Test that ``LazyRfc3339UtcTime`` is rendered as ``str`` using UTC timestamp."""
+    """Test ``LazyRfc3339UtcTime`` renders as ``str`` UTC timestamp."""
```

Comment:
I just realized that this is also changed in #2008 which would be conflicting.

While not necessarily related to this PR, I think it's fine to keep it at a part of this patch and won't demand a dedicated PR.

Reply:
I just pulled this out of 2008. I'll let this make the update instead since it's relevant.